File: DS-Algo-with-Java/Lecture-10/CountVowel.java
import java.util.Scanner;

public class CountVowel {
    public static void main(String[] args) {
        try (Scanner sc = new Scanner(System.in)) {
            System.out.print("Enter the string: ");
            String s = sc.nextLine();
            System.out.print(count(s));
        }
    }

    public static int count(String s) {
        int count = 0;
        for(int i = 0; i < s.length(); i++) {
            char ch = s.charAt(i);
            if(ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u'){
                count++;
            }
        }
        return count;
    }
}

File: DS-Algo-with-Python/Lecture-10/CountVowel.py
def count(s):
    count = 0
    for i in range(len(s)):
        ch = s[i]

        if ch == 'a' or ch == 'e' or ch == 'i' or ch == 'o' or ch == 'u':
            count += 1
        
    return count

s = input("Enter the string: ")
print(f"Vowel = {count(s)}")
